### ISSUE TITLE
TypeError: forever.config.saveSync is not a function

### ISSUE THREAD

**Issue by akatona84** [foreversd/forever#1114]
4.0.0 introduced an issue, tried to call `forever columns reset` but I'm assuming it would fail everywhere where this saveSync is used.
#1112 changed the config's type to `Configstore` from `nconf`, imho some further adjustments are needed.

```
  forever.config.saveSync();
                 ^

TypeError: forever.config.saveSync is not a function
    at Object.app.cmd.cli.resetColumns (/root/nodejs/node_modules/node/node_modules/forever/lib/forever/cli.js:590:18)
    at apply (/root/nodejs/node_modules/node/node_modules/director/lib/director/router.js:448:19)
    at _every (/root/nodejs/node_modules/node/node_modules/director/lib/director/router.js:30:9)
    at exports.Router.Router.invoke (/root/nodejs/node_modules/node/node_modules/director/lib/director/router.js:454:5)
    at exports.Router.Router.dispatch (/root/nodejs/node_modules/node/node_modules/director/lib/director/cli.js:63:8)
    at /root/nodejs/node_modules/node/node_modules/flatiron/lib/flatiron/plugins/cli.js:87:18
    at exports.App.App.init (/root/nodejs/node_modules/node/node_modules/broadway/lib/broadway/app.js:73:12)
    at exports.App.app.start (/root/nodejs/node_modules/node/node_modules/flatiron/lib/flatiron/plugins/cli.js:82:9)
    at /root/nodejs/node_modules/node/node_modules/forever/lib/forever/cli.js:645:9
    at onComplete (/root/nodejs/node_modules/node/node_modules/broadway/lib/broadway/app.js:85:5)
```

**Comment by akatona84** [comment]
Hello @kibertoad , could you take a look at this issue, please? This is quite critical on our side to be fixed, and having a release with the fix.

We specified the 3.0.4 version to be used wherever we could for active builds of ours, but whenever we're creating a hotfix/patch release, the latest forever is pulled in, where this issue is broken.

**Comment by kibertoad** [comment]
Fix was released in 4.0.1
